### ISSUE TITLE
Multiple width views in single template

### ISSUE THREAD

**Issue by benjamincoleman** [devinit/DIwebsite-redesign#816]
We want to have 3 'windows' in the single page to show the visualisation at 3 different screen widths so that the editor is able to see the three versions of the chart breakpoints. 

1. Mobile
2. Middle
3. Desktop

You can see sketches for this here: 

https://marvelapp.com/prototype/3j51290/screen/70492428

**Comment by BenDarby** [comment]
@benjamincoleman so will we just mimic the 3 breakpoints by attaching a modifier class to each version to control the width and include only that size image so they can see how each one works?

**Comment by benjamincoleman** [comment]
@BenDarby exactly that - the purpose is so that they can preview the options that they have set up for fallback images.

**Comment by BenDarby** [comment]
@benjamincoleman I have set up so the desktop chart is the full visualisation like normal, then the 2 versions below are just showing their fallback image, the size for mobile is up to 500px and tablet is up to 700px, I've hidden the interactive chart for these so they can see how the image looks. Is this correct? https://di-vis.surge.sh/page-templates/26-interactive-visualisation

**Comment by benjamincoleman** [comment]
@BenDarby thanks for this. 

Can you pull out the "desktop / tablet / mobile" text to be separate from the design - it gives them something to reference. 

> I've hidden the interactive chart for these so they can see how the image looks.

So, we'll want this controlled by the CMS, so please make sure there's a class we can switch on and off with code to declare whether we want to show or hide the interactive chart. This would be on both tablet and mobile instances.

**Comment by BenDarby** [comment]
@urlsangel @benjamincoleman this has been updated now, we have a class of `chart-tablet` and `chart-mobile` applied to parent containers, these now just control the width. And we now have a class of `show-fallback` that can be applied to the same container that just toggles the showing and hiding of a fallback image/visual charts, this class will most likely always be applied to the mobile chart and optional for the tablet chart but it can also be an optional class for mobile just in case.

I also added labels to the right of each section to signal which view is which.
